### PR TITLE
suggest input component

### PR DESCRIPTION
**Pull request by a-type**
gif

### CODE REVIEW

File: src/components/SearchBox/SearchBox.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    return (
+      <Wrapper>
+        <Autosuggest
+          {...this.props}
+          inputProps={{
+            ...inputProps,
+            onChange,
+            value,
+            onFocus: (a, b, c, d, e) => {
+              console.log('focus');
```
Comment (tato123) [inline src/components/SearchBox/SearchBox.js]: Is the console log still needed?

File: src/components/SearchBox/SearchBox.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+              if (inputProps.onFocus) {
+                inputProps.onFocus(a, b, c, d, e);
+              }
+              if (this.props.showSuggestionsOnFocus) {
+                this.setState({ forceShowSuggestions: true });
+              }
+            },
+            onBlur: (a, b, c, d, e) => {
+              console.log('blur');
```
Comment (tato123) [inline src/components/SearchBox/SearchBox.js]: Is this log statement still needed as well?

**Comment by a-type** [comment]
Sorry about the logs, this is still a bit WIP. I'll mention you when I'm done splitting it out.

**Comment by a-type** [comment]
@tato123 should be ready to go now

**Comment by a-type** [comment]
Also fixes #38

**Pull request merged**